### ISSUE TITLE
Barriers & Gates

### ISSUE THREAD

**Issue by YGMZ** [IsraelHikingMap/Map#159]
Presently thereare 2 types of gates (closed / open).
3 types of gates required:
1 - open for motor vehicle (practically open for all...)
defined by: acsess all = yes or acsess motor vehicle = yes
2 - open for biking or foot
on IHM defined by: acsess foot = yes and acsess motor vehicle = no
on IMTBM defined by: acsess bicycle = yes and acsess motor vehicle = no
3 - closed

Presently 1 type of barrier.
2 types required:
1 - open for biking or foot
on IHM defined by: acsess foot = yes and acsess motor vehicle = no
on IMTBM defined by: acsess bicycle = yes and acsess motor vehicle = no
2 - closed
Note: open for all not required because in this case it is not prectically a barrier...

**Comment by zstadler** [comment]
I agree that differentiation by access is important. This was also the intention. 

The intention for a closed gate icon in the Hiking map is to capture all barriers with no access, even by foot, regardless of the barrier type. 

Similarly, the block icon intends to capture all barriers with no access to motor vehicles, regardless of barrier type. 

Other barriers are expected to be open for all, and have different icons by type: gate, lift gate, stile, cattle grid.

The legend needs to reflect these intentions. For example, "No Access" instead of "Closed gate" and "No Vehicles" instead of "Block". 

Issue #158 may indicate a bug in implementing the above intentions...

**Comment by YGMZ** [comment]
I would not mix icons between gates and other barriers - gates are unique and deserve their own 3 icons (they are part of a fence in many cases, and may also surprise a hiker/biker by being closed....)
Notes:
(1) IMTBM lacks barrier in the legend.
(2) IHM has block in the legend (block is a type of barrier...)

**Comment by zstadler** [comment]
> I would not mix icons between gates and other barriers - gates are unique and deserve their own 3 icons (they are part of a fence in many cases, and may also surprise a hiker/biker by being closed....)

A gate icon is used for both fully accessible and fully inaccessible barriers. No surprise is possible here. 

The only possible surprise is a gate accessible to hikers/bikers and closed to vehicles. It will show as a block icon. However, I don't see much room for surprises here:

I can only recall such gates as horizontal metal bars which are set-up by farmers, nature reserve authority, etc., to prevent unauthorized vehicles from entering a private dirt road. 

These are closed gates that cannot be further closed. Access to hikers and bikers can only be prevented by replacing the gate with a different type of gate, which is not different than adding or removing a barrier, in requiring a map update.

Note: there is also a risk of confusion by having 3 icons that show a gate with even smaller differences between them than the difference between the existing 2 icons.

**Comment by YGMZ** [comment]
I agree that in order to keep it simple 3 icons are enough.
presently IMTBM has only 2.
I suggest:
(1) to add the following icon for a gate that in which access is possible for vehicle (as well as hiker/rider).
         https://drive.google.com/file/d/0BwKVGChbqrLtandqeGpGTVUyWGs/view?usp=sharing 
(2) Legend should be ammended as well

**Comment by YGMZ** [comment]
I agree that in order to keep it simple 3 icons are enough.
presently IMTBM has only 2.
I suggest:
(1) to add the following icon for a gate that in which access is possible for vehicle (as well as hiker/rider).
https://drive.google.com/file/d/0BwKVGChbqrLtNkVpOG81N3A5cFk/view?usp=sharing
(2) Legend should be ammended as well

**Comment by YGMZ** [comment]
I suggest to ammend the legand according to attched JPG file above
(1) Gate/Barrier closed to vehicle, open to foot/bicycle
(2) Gate/Barrier open to all
(3) Gate/Barrier closed to all

**Comment by zstadler** [comment]
Issue #160 is "Unify icons between MTB and IHM". I think the 3 barrier classes are good candidates for such unification.

As for the legend, luckily, the legend is taken directly from the map. In fact, the legend also works for private base maps. For example, this is the legend for standard OSM map ( Mapnik):

![image](https://user-images.githubusercontent.com/1304610/30241781-dfa6bab2-9592-11e7-8f6c-f3313cb7485d.png)

**Comment by YGMZ** [comment]
I agree on the 3 types of barriers/gates for the unification.
I suggest to eliminate the cattle grid - (1) it is just another type of barrier (2) usualy it has no impact/significance on drivers/riders/hikers. if it does have an impact, let it be marked as one of the 3 classes of barriers (we do not want to have 2 or 3 type of cattle grids.....)

**Comment by zstadler** [comment]
Cattle grid and lift gate are descriptive versions of an open gate. Stile/kissing gate is a descriptive version of vehicle block. They have a different icon primarily for orientation purposes..

**Comment by zstadler** [comment]
@valleyofdawn 
Do you agree to apply this proposal also to the MTB map?

**Comment by valleyofdawn** [comment]
I agree that we should use the same typology as IHM, but I don't see the utility of a third type of gate for bikers. Cattle grids are useful as waymarks in KKL tracks.

**Comment by YGMZ** [comment]
The 3rd gate (access=yes to foot & bike & vehicle) gives info as to where one can reach by vehicle and from there on continue by bike, or in case one needs vehicle "rescue" (bike mechanical failure or self....)

**Comment by valleyofdawn** [comment]
Same can be said on any vehicle specific feature, such as 4 levels of track difficulty. Users can consult IHM when using a car.
On the other hand, it complicates the map. The current icons are self-explanatory, and alternative ones might not be.

**Comment by zstadler** [comment]
@valleyofdawn, I lost you...

Please clarify which of the following 3 barrier types you "don't see the utility of ... for bikers":
(1) Gate/Barrier closed to vehicle, open to foot/bicycle
(2) Gate/Barrier open to all
(3) Gate/Barrier closed to all

Does it mean you object to showing it on the Bike map?

Also, bikers do walk from time to time and use cars to get around...

**Comment by valleyofdawn** [comment]
I don't see the utility of differentiating 1 and 2 in the bike map.
When bikers drive or walk they should not use a bike map.

**Comment by zstadler** [comment]
IMO, a user, such as a biker, wants to use one map to plan his entire trip, including the surrounding.
If I'm not mistaken, the issue was opened by a biker.

In any case, if despite the above, you agree to have (1) in bike map it will serve the consistency of the maps.

**Comment by valleyofdawn** [comment]
Are there proposed icons? If they are as self explanatory as the current ones I can live with that.

**Comment by zstadler** [comment]
The is the current "Vehicle Block" icon in the Hiking map. I think it is self-explanatory.
![image](https://user-images.githubusercontent.com/1304610/30952365-41449ffe-a430-11e7-87f8-8566902e5238.png)

**Comment by YGMZ** [comment]
The vehicle block we already have in the MTB map - it is item: (1) Gate/Barrier closed to vehicle, open to foot/bicycle. Also existing is Item: (3) Gate/Barrier closed to all

What is missing is item: (2) Gate/Barrier open to all
Adding (2) will also serve consistancy with IHM

My proposed icon is in the following link - it is well within the "family" of the 2 existing icons and self explainatory:
https://drive.google.com/file/d/0BwKVGChbqrLtNkVpOG81N3A5cFk/view?usp=sharing

**Comment by zstadler** [comment]
The open gate icon in the MTB map
![image](https://user-images.githubusercontent.com/1304610/30989725-5c7d7f94-a4a7-11e7-96c3-2b9eafc4895a.png)
represents both (1) and (2): Gate/Barrier open to foot/bicycle, regardless vehicle access permission. I suggest it continues to represent (2) Gate/Barrier open to all.

The difference between an open 1-door gate and an open 2-door gate is not so clear to me. The bollards icon used in IHM, on the other hand, IMO represent well a barrier that is applicable only to vehicles.

**Comment by YGMZ** [comment]
I think different:
I see one gate which is half/semi open and a slot is left for hikers/bikers, and another gate with its 2 wings pointing out wards and is wide open - very intuitive for me
I would not mix gates and bollards icons - it gives the impression the map is differantiating gate & bollards/other barriers while it is not (as we want to keep it simple)
I suggest asking some more users.

**Comment by valleyofdawn** [comment]
I'm with Yariv on this one.
Only problem is that the new icon is very wide.

**Comment by YGMZ** [comment]
With the new icons barriers & gates look diff in IHM Vs IMTBM, see picture below.
I suggest:
(1) having a 3rd icon (colored 1/2 black and 1/2 red) defined as open to bicycle and hiker & closed to motor vehicle.
(2) unifying definitions on both maps: black gate = open to all, red gate = closed to all

![gates barriers shown different on ihm vs imtbm](https://user-images.githubusercontent.com/31124242/48318695-ea516900-e60c-11e8-8c2d-626a67562f1e.jpg)

**Comment by valleyofdawn** [comment]
You are right, we should look into our gate definitions and standardize them.
